streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -439,7 +439,7 @@
                     RECONNECT_BACKOFF_MS_DOC)
             .define(RECONNECT_BACKOFF_MAX_MS_CONFIG,
                     ConfigDef.Type.LONG,
-                    50L,
+                    1000L,
```

Comment:
I'm not sure of the original motivation behind setting it to `50`. But if we are going to define it as `1000` i think it might be better to remove this and `reconnect_backoff_min_ms_config`  definitations from here as they are the same as what is set in the `ProducerConfig` and `ConsumerConfig`, so it seem pretty pointless overriding them